pyerrors/correlators.py

```diff
@@ -868,7 +865,7 @@ def set_prange(self, prange):
         self.prange = prange
         return
 
-    def show(self, x_range=None, comp=None, y_range=None, logscale=False, plateau=None, fit_res=None, fit_key=None, ylabel=None, save=None, auto_gamma=False, hide_sigma=None, references=None, title=None):
+    def show(self, x_range: Optional[list[int]]=None, comp: Optional[Corr]=None, y_range: None=None, logscale: bool=False, plateau: None=None, fit_res: Optional[Fit_result]=None, fit_key: Optional[str]=None, ylabel: None=None, save: None=None, auto_gamma: bool=False, hide_sigma: None=None, references: None=None, title: None=None):
```

Comment:
The types of `y_range`, `plateau`, `ylabel`, `save`, `hide_sigma`, `references`, `title` have not been recognized.

Reply:
When a type has not been recognised, this indicates that we don't have proper test coverage for this call 😬